### PR TITLE
Changes to compile on fleet master branch. [Ready]

### PR DESCRIPTION
**Pull request by zeisss**
Make our fleet-client compatible with fleet 0.7

### CODE REVIEW

File: client_api.go [hunk 1]
```diff
@@ -95,7 +90,7 @@ func (this *ClientAPI) Start(name string) error {
 	if err != nil {
 		return errgo.Mask(err)
 	}
-	this.client.SetJobTargetState(u.Name, job.JobStateLaunched)
+	this.client.SetUnitTargetState(u.Name, STATE_LAUNCHED)
```
Comment (xh3b4sd) [inline client_api.go:93]: From where does `STATE_LAUNCHED` come from?

File: client_api.go [hunk 2]
```diff
@@ -198,10 +188,20 @@ func (this *ClientAPI) getMachineIp(unitName string) (string, error) {
...
-		if machine.ID == su.TargetMachineID {
+		if machine.ID == unit.MachineID {
 			ip = machine.PublicIP
 		}
 	}
 
 	return GetMachineIP(ip), nil
 }
+
+func description(options []*schema.UnitOption) string {
```
Comment (xh3b4sd) [inline client_api.go:199]: Does a `Unit` no longer provide the `Description()` method?

Comment (zeisss) [inline client_api.go:199]: Correct.

File: client_api.go (hunk 1 above)
Comment (zeisss) [inline client_api.go:93]: Our `client.go` - I could have also cast their `jobStateLaunched` back into a string, but this is a dependency less on their code and we already have keep our constants in sync.

**Comment by xh3b4sd** [comment]
I dont like the api implementation of the client, but this is out of the scope of that PR. LGTM

**Pull request merged**